mocap_base/include/mocap_base/MoCapDriverBase.h

```diff
@@ -12,7 +12,7 @@
 #include "rclcpp/rclcpp.hpp"
 #include <nav_msgs/msg/odometry.hpp>
 #include <geometry_msgs/msg/pose_stamped.hpp>
-#include <tf2_eigen/tf2_eigen.h>
+#include <tf2_eigen/tf2_eigen.hpp>
```

Comment:
would people need to update their tf2 eigen for this file to exist or does it already exist in current humble setups?